pkg/tinydtls/Makefile.include

```diff
@@ -5,6 +5,9 @@ INCLUDES += -I$(PKG_BUILDDIR)
 ifeq ($(shell uname -s),Darwin)
     CFLAGS += -Wno-gnu-zero-variadic-macro-arguments -Wno-unused-function
 endif
+ifneq (,$(filter $(TOOLCHAIN),llvm clang))
```

Comment:
you don't need to filter, there's an override in Makefile.include which replaces TOOLCHAIN=clang with TOOLCHAIN=llvm if the user sets it.

```make
ifeq ($(TOOLCHAIN),llvm)
...
endif
```